src/globals/scss/_helper-classes.scss

```diff
@@ -16,3 +16,16 @@
   white-space: nowrap;
   overflow: hidden;
 }
+
+.#{$prefix}--assistive-text {
+  position: absolute;
```

Comment:
Seems to have the same content as `.bx--visually-hidden`. Do you want to make one alias of another, or do you want to simply use `.bx--visually-hidden`?